### PR TITLE
Makes Profile independent from Measure

### PR DESCRIPTION
**Pull request by oterral**
Changing the design of the draw functionnality implies a little refactoring of Profile and Measure  directives. 

With this PR the Profile accepts an ol Feature as parameter, listen every change on this feature and redraw the profile automatically.

This PR makes also the width of the profile dynamic. No need to specify a width by default anymore, the Profile takes the size of the parent div.

[TEST](https://mf-geoadmin3.dev.bgdi.ch/teo_prof/)

### CODE REVIEW

File: src/components/profile/ProfileService.js [hunk 1]
```diff
@@ -260,10 +278,19 @@
...
-      var d3LibUrl = this.d3libUrl;
       return function(options, lazyLoadCB) {
-        return new ProfileChart(options, lazyLoadCB, d3LibUrl);
+        // Lazy load of D3 library
+        var onD3Loaded = function() {
+          d3 = $window.d3;
+          lazyLoadCB();
+        };
+        if (!$window.d3) {
+          window.console.log('la');
```
Comment (gjn) [inline src/components/profile/ProfileService.js]: debug code...

File: src/js/MeasureController.js [hunk 2]
```diff
@@ -23,7 +22,8 @@
                 left: 60
               },
               height: 188,
-              elevationModel: 'COMB'
+              elevationModel: 'COMB',
+              profileFeature: null
```
Comment (gjn) [inline src/js/MeasureController.js]: can't see where profileFeature is needed. I don't think it is.

**Comment by gjn** [comment]
Very nice re-factoring. Added 2 minor comments, otherwise, I think this can be merged.

**Comment by oterral** [comment]
Thx @gjn I fix the 2 issue and then merge

**Pull request merged**